### PR TITLE
Fix/added scroll view for keyboard

### PR DESCRIPTION
**Pull request by bmorris-revacomm**
- added scroll view so keyboard doesn't block textfields (there are a couple little UI bugs that I am looking at fixing - they don't affect the functionality at all)
- return key on keyboard now dismisses keyboard
- created openPopUp function to clean up code
- deleted Aircrew storyboard

### CODE REVIEW

File: Logging/Controllers/View Controllers/Form 781/ContainerViews/MissionDataViewController.swift [hunk 1]
```diff
@@ -81,8 +81,31 @@ class MissionDataViewController: UIViewController {
         flightSeqTableView.delegate = self
         flightSeqTableView.dataSource = self
         dateTextField.delegate = self
+<<<<<<< HEAD
```
Comment (petermisik) [inline Logging/Controllers/View Controllers/Form 781/ContainerViews/MissionDataViewController.swift:84]: You have a conflict header here you need to resolve.

**Review by petermisik — CHANGES_REQUESTED** [review]
If you resolve the conflict, I'm good with everything else.

File: Logging/Controllers/View Controllers/Form 781/ContainerViews/MissionDataViewController.swift (hunk 1 above)
Comment (bmorris-revacomm) [reply Logging/Controllers/View Controllers/Form 781/ContainerViews/MissionDataViewController.swift]: Yes, I saw that and just fixed it!

**Review by petermisik — APPROVED** [review]
Everything looks good.

**Pull request merged**